tests/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ if(MSVC)
 
 	# Add linker flags
 	set(CMAKE_EXE_LINKER_FLAGS /DEBUG)
+else()
+	if(CMAKE_SYSTEM_PROCESSOR MATCHES "x86" OR CMAKE_SYSTEM_PROCESSOR MATCHES "amd64")
```

Comment:
Ideally, `CMAKE_SYSTEM_PROCESSOR` should be first made lower-case, but most systems have it set to lowercase anyway